shared/CloudBackupAndRestoreUtils.ts

```diff
@@ -413,8 +480,8 @@ class Cloud {
 export default Cloud;
 
 export type ProfileInfo = {
-  email: string;
-  picture: string;
+  email: string | undefined | null;
+  picture: string | undefined;
```

Comment:
For iOS we are setting picture as null, should we also in type null here?
`picture: string |null`

Reply:
done